### PR TITLE
feat: apply new branding guidelines

### PR DESCRIPTION
**Pull request by Veenders**

### CODE REVIEW

File: src/components/MaTextField/MaTextField.css [hunk 1]
```diff
@@ -37,11 +38,16 @@
...
-  background-color: var(--color-brown-light);
+  background-color: var(--color-gray-lighter);
   color: var(--color-gray-base);
   pointer-events: none;
 }
 
+.ma-text-field__input-wrapper > .ma-button {
+  border-top-right-radius: var(--radius-small);
+  border-bottom-right-radius: var(--radius-small);
+}
```
Comment (icurdu) [inline src/components/MaTextField/MaTextField.css:48]: The button is displaying weird.
    ```
border-bottom-left-radius: var(--radius-none);
    border-top-left-radius: var(--radius-none);
    padding: 0.5rem;
```

File: src/tokens/headings.ts [hunk 2]
```diff
@@ -1,76 +1,111 @@
...
   },
-  small: {
-    'font-size': '1.75rem',
-    'line-height': 1.15,
-    'top-crop': '-0.2644em',
-    'bottom-crop': '-0.2214em',
+  large: {
+    'font-size': '3.5rem',
+    'line-height': 1.05,
+    'letter-spacing': '-2px',
```
Comment (icurdu) [inline src/tokens/headings.ts:15]: change letter-space to -1.5px on desktop-medium

File: src/tokens/headings.ts [hunk 3]
```diff
@@ -1,76 +1,111 @@
...
-    'top-crop': '-0.2399em',
-    'bottom-crop': '-0.1969em',
-  },
-  large: {
-    'font-size': '2.5rem',
+    'font-size': '3rem',
     'line-height': 1.1,
-    'top-crop': '-0.243em',
-    'bottom-crop': '-0.2em',
+    'letter-spacing': '-2px',
```
Comment (icurdu) [inline src/tokens/headings.ts:23]: change letter-space to -1.5px on desktop-medium

File: src/tokens/headings.ts [hunk 4]
```diff
@@ -1,76 +1,111 @@
...
   },
-  xlarge: {
-    'font-size': '2.5rem',
-    'line-height': 1.1,
-    'top-crop': '-0.243em',
-    'bottom-crop': '-0.2em',
+  small: {
+    'font-size': '2rem',
+    'line-height': 1.15,
+    'letter-spacing': '-2px',
```
Comment (icurdu) [inline src/tokens/headings.ts:31]: change letter-space to -1px on desktop-medium

File: src/tokens/headings.ts [hunk 5]
```diff
@@ -1,76 +1,111 @@
...
   },
-}
-
-const mediumScreenHeadingSize = {
   xsmall: {
     'font-size': '1.5rem',
     'line-height': 1.15,
-    'top-crop': '-0.25em',
-    'bottom-crop': '-0.2em',
+    'letter-spacing': '-2px',
```
Comment (icurdu) [inline src/tokens/headings.ts:39]: change letter-space to -0.75px on desktop-medium

File: src/components/MaAlert/MaAlert.css [hunk 6]
```diff
@@ -48,21 +44,17 @@
...
 }
 
 .alert--yellow a {
-  color: var(--color-pink-base);
-}
-
-.wrapper {
-  margin-top: 0.2rem;
+  color: var(--color-orange-dark);
 }
```
Comment (icurdu) [inline src/components/MaAlert/MaAlert.css:60]: Can we change the stack gap to 0.75rem inside the alert vue component?

**Review by icurdu — CHANGES_REQUESTED** [review]
We also need to change Ma-Link component in order to match ma-button styles

**Review by alex-castelo — APPROVED** [review]

**Pull request merged**